test/unit/option_spec.lua

```diff
@@ -0,0 +1,61 @@
+local helpers = require("test.unit.helpers")
+
+local to_cstr = helpers.to_cstr
+local eq      = helpers.eq
+
+local option = helpers.cimport("./src/nvim/option.h")
+local globals = helpers.cimport("./src/nvim/globals.h")
+
+local check_ff_value = function(ff)
+  return option.check_ff_value(to_cstr(ff))
+end
+
+local get_sts_value = function()
+  return option.get_sts_value()
+end
```

Comment:
Since there is nothing being done in this function I would not define a local function for it. Just call `option.get_sts_value()` directly. 

Even cases like `option.check_ff_value(to_cstr(ff))`, to me, are not worth abstracting, though I know many of our tests do so.